beacon_node/eth1/src/service.rs

```diff
@@ -301,9 +316,22 @@ impl Service {
     pub async fn update(
         &self,
     ) -> Result<(DepositCacheUpdateOutcome, BlockCacheUpdateOutcome), String> {
+        let endpoint = &self.config().endpoint.clone();
+        let remote_head_block_number =
```

Comment:
We could save a call to the eth1 node by using eth_getBlockByNumber with 'latest' in the params to get the block corresponding to remote head.

Reply:
Nice one, I'll do this!

Reply:
Done here: https://github.com/sigp/lighthouse/pull/1797/commits/b0d037375eeab42d05e1a5653da0e56ac276ce60

:)